javascript-methods/javascript-methods-quiz-notes.md

```diff
@@ -15,11 +15,13 @@ array.pop()
 - How do you round a number down to the nearest integer?
 math.floor()
 - How do you generate a random number?
-math.floor(math.random(array.length-1))
+math.random(array.length-1)
```

Comment:
You don't need to pass in `array.length - 1`. You would just call the `Math.random()` method directly.